README.md

```diff
@@ -19,9 +19,9 @@ Axom provides robust, flexible software infrastructure for the development of mu
 Documentation
 -------------
 
-Latest docs on Develop branch: https://axom.readthedocs.io
+To access the latest docs for Axom: https://axom.readthedocs.io/en/develop/
 
-To access docs for other versions: https://readthedocs.org/projects/axom/
+To access Axom docs for specific versions: https://app.readthedocs.org/projects/axom/
```

Comment:
I am not sure how much value the second link is giving us much. They have to know to click on the book icon not the word "develop". It really isn't a list of versions, it is more of a list of builds, which a user doesn't care about.

This is just my opinion for what little it's worth.

Reply:
If we don't have the second link, how does one get to a specific version?

Reply:
It's in the bottom right hand corner

![image](https://github.com/user-attachments/assets/ede2645e-d4aa-4d57-b3a4-19abb12fe300)

Reply:
Ah. Is that clearer? It obviously wasn't for me.

Reply:
That "develop" button is handy. I didn't know about that.